docs/Files/UI/Solution-Manager/Library/Departments/Departments.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
this file should be named Managing-Departments.md

Reply:
Adjusted file name to be Managing-Departments.md